lib/filesystem/CMakeLists.txt

```diff
@@ -103,3 +103,4 @@ install(
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/irods/filesystem"
   COMPONENT ${IRODS_PACKAGE_COMPONENT_DEVELOPMENT_NAME}
 )
+
```

Comment:
I see you've tried to remove this blank line twice and it's still here. I think you might be fighting your IDE or text editor here.

Close any open instances you have of the file in your IDE or text editor and then run the following command to get git to do the work for you:
```
git reset origin/main lib/filesystem/CMakeLists.txt
```

Reply:
Oddly enough, I didn't see a line in my copy of CMakeLists and after running that command my working tree is still clean.

Reply:
In that case, wait until your commits are squashed before clicking resolve on this. Let's see if it goes away on its own.

Reply:
I just checked the branch on Github and that line isn't there. Success?

Reply:
No, the file is still showing as modified in your PR. Specifically, in the first commit of your PR. We're not done here yet.

Reply:
You got it!

Reply:
You can mark this as resolved now